packages/core-js-compat/src/data.mjs

```diff
@@ -1206,6 +1206,7 @@ export const data = {
     safari: '11.0',
   },
   'es.promise.with-resolvers': {
+    android: '126',
```

Comment:
I checked it on Android Chrome 123 and this exists, so it seems MDN data is incorrect.

Reply:
MDN also shows M119:
![image](https://github.com/zloirock/core-js/assets/2213682/d8f220f3-2f93-4131-9f19-2a2fdd8783d2)

Reply:
I understand. It's a `caniuse` issue (that's, for example, partially inherited in `browserslist`). That stores data only for the latest mobile browsers releases, so even for old features you will see that they are supported only from Android M126 Chrome / WebView.

![image](https://github.com/zloirock/core-js/assets/2213682/5aab886f-4f93-4727-85cd-1efc6aa3e413)

So I think it can be closed.

Reply:
![image](https://github.com/zloirock/core-js/assets/1017440/e1fa7c78-415a-4731-b61c-1b93d040aa7f)

This is what I am seeing in the wild

I got confused as first as the short hand for Chrome Mobile iOS was "Chrome Mobile"

Reply:
Chrome on iOS is just a wrapper for WebView -> Safari.